src/sentry/testutils/helpers/features.py

```diff
@@ -44,6 +46,20 @@ def Feature(names):
 
     def features_override(name, *args, **kwargs):
         if name in names:
+            if name in sentry.features.UNSCOPED_FEATURES:
+                return names[name]
+
+            if name.startswith("organizations:"):
+                # org = kwargs.get("organization", None)
+                org = args[0] if len(args) > 0 else kwargs.get("organization", None)
+                if not isinstance(org, Organization):
+                    raise ValueError("Must provide organization to check feature")
+
+            if name.startswith("projects:"):
```

Comment:
I think that what we could do here is rather than relaying on the name we can fetch the feature. We can do this like
```
from sentry import features
feature = features.get(name)
```

That has a couple of benefits:
 - If the feature is not registered at all, then it will throw a `FeatureNotRegistered` exception. That'll help us catch any typos in feature names
 - If the feature is registered, then we can check whether it is an `OrganizationFeature`, `ProjectFeature` or `Feature` (ie unscoped feature) and check for the appropriate params that way. That also allows us to remove `UNSCOPED_FEATURES`